### DISCUSSION TITLE
curl: (56) Recv failure: Connection reset by peer

### DISCUSSION THREAD

**Discussion by carlosd-29** [edgexfoundry/community#11] [Q&A]
Going through this quick start guide for jakarta: https://docs.edgexfoundry.org/2.1/getting-started/quick-start/ and I can see that everything is up and running:
![image](https://user-images.githubusercontent.com/70228341/189749926-e8ad99e8-a486-4d61-aba3-027ca2cb5fff.png)
But when I try to check if virtual device is connected properly with:
`curl http://localhost:59880/api/v2/event/device/name/Random-Integer-Device`
I get the following error:
` curl: (56) Recv failure: Connection reset by peer`

**Comment by carlosd-29** [comment]
Suspected this might have been due to running on CentOS or some issue with my environment so I ran it through a docker container with Ubuntu:20.04 and that worked fine

**Reply by lenny-goodell** [reply]
I work with @carlosd-29 on this and the issue was all the EdgeX services where restarting because they couldn't connect to Consul, which kept terminating for some unknow reason. 

Anytime we see `(56) Recv failure: Connection reset by peer` it means the service is restarting due to start-up issues. Need to dig through the logs to root cause why.
